corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/bootstrap5/sub_group.html

```diff
@@ -2,6 +2,16 @@
 {% load hq_shared_tags %}
 
 <script type="text/html" id="sub-group-fullform-ko-template">
+  <!-- ko if: isDummy -->
+  <div style="margin-bottom: 10px; margin-right: 10px">
```

Comment:
This is the B3 markup, not the B5 markup.

Reply:
thanks. good catch